src/components/FeatureViews/Registration/RegistrationCart/PaymentBoard.js

```diff
@@ -60,400 +64,479 @@ const CREATE_ENROLLMENTS = gql`mutation CreateEnrollments($enrollments:[Enrollme
+                                        control={<Radio />}
+                                        data-cy={`${label}-checkbox`}
+                                        key={value}
+                                        label={label}
+                                        value={value} />
+                                ))
+                            }
+                        </RadioGroup>
+                    </FormControl>
+                </Grid>
+                <Grid item>
+                    <Table>
+                        <TableRow className={classes.root}>
+                            <TableCell>Sub-Total</TableCell>
+                            <TableCell>$ {priceQuote.subTotal}</TableCell>
+                        </TableRow>
+                        <TableRow className={classes.root}>
+                            <TableCell>Account Balance</TableCell>
+                            <TableCell>$ {priceQuote.accountBalance}</TableCell>
+                        </TableRow>
+                        <TableRow className={classes.root}>
+                            <TableCell>Discounts</TableCell>
+                            <TableCell> </TableCell>
+                        </TableRow>
+                        <TableRow>
+                            <TableCell>Price Adjustment</TableCell>
+                            <TableCell style={{"width": "55%"}}>
+                                $ - <TextField
+                                    inputProps={{
+                                        "style": {
+                                            "padding": 8,
+                                            "textAlign": "center",
+                                        },
+                                    }}
+                                    onChange={handlePriceAdjustmentValueChange}
+                                    style={{"width": "30%"}}
+                                    type="number"
+                                    value={priceAdjustment}
+                                    variant="outlined" />
+                            </TableCell>
+                        </TableRow>
+                        <TableRow className={classes.root}>
+                            <TableCell>Total</TableCell>
+                            <TableCell>$ {priceQuote.total}</TableCell>
+                        </TableRow>
+                    </Table>
+                </Grid>
+            </Grid>
+            <Grid container item justify="flex-end">
+                <Grid item>
+                    <ResponsiveButton
+                        color="primary"
+                        data-cy="pay-action"
+                        disabled={paymentMethod === null || priceQuote.total === "-" || priceQuote < 0}
+                        onClick={handlePayment}
+                        variant="contained">
+                        {paymentMethod === "credit_card" ? "Pay Now" : "Pay Later"}
```

Comment:
this doesn't follow Gaby's design - we're supposed to enable/disable buttons depending on payment methods. not completely switch them out. Also I'm assuming the pay later flow hasn't been implemented?
<img width="1022" alt="Screen Shot 2020-12-13 at 2 20 58 PM" src="https://user-images.githubusercontent.com/12959959/102025680-7daa7a80-3d4e-11eb-8abe-19ba0702077d.png">

<img width="1017" alt="Screen Shot 2020-12-13 at 2 21 12 PM" src="https://user-images.githubusercontent.com/12959959/102025682-813e0180-3d4e-11eb-8981-b4c10d61895a.png">

Reply:
yeah, i havent gotten around to pay later yet